crates/symbolicator/src/services/download/sentry.rs

```diff
@@ -205,10 +205,19 @@ impl SentryDownloader {
                 .append_pair("debug_id", &debug_id.to_string());
         }
 
+        // Minidumps on Windows are assigned an object type of ObjectType::Pe, but it's possible
+        // that they're actually Elfs instead
+        let additional_filetype = if object_id.object_type == ObjectType::Pe {
+            Some(FileType::ElfDebug)
+        } else {
+            None
+        };
+
```

Comment:
i fear this is the wrong place to fix this up, i expect it should be fixed in whoever calls this function to pass the right slice of `FileType`s.  but still looking for all the places where this could be caused

Reply:
https://github.com/getsentry/symbolicator/blob/master/crates/symbolicator/src/sources.rs#L462 might be closer to where this should be done

Reply:
In that case I think https://github.com/getsentry/symbolicator/blob/master/crates/symbolicator/src/services/symbolication.rs#L1716 might also be an option. This is assuming we're only seeing this issue with minidumps, though.

Reply:
I don't think so, `MinidumpState::object_type` is only used in https://github.com/getsentry/symbolicator/blob/87e846b006971167ff1d5af574272cfa0163158d/crates/symbolicator/src/services/symbolication.rs#L1878 which is not used in any code paths ending up calling `.list_files()`.  The other location is the one which ends up being used in `.list_files()`.